header/p2p/server.go

```diff
@@ -139,5 +160,9 @@ func (serv *ExchangeServer) handleRequest(from, to uint64, stream network.Stream
 			stream.Reset() //nolint:errcheck
 			return
 		}
+
+		if err = stream.SetWriteDeadline(time.Time{}); err != nil {
```

Comment:
Same here, whether we actually need this.

Reply:
the same as above. libp2p does not reset timer per write. So, we have to do it manually

Reply:
Yes, this resetting is required

Reply:
ack.